app/static/js/userProfile.js

```diff
@@ -128,7 +128,13 @@ $(document).ready(function(){
             return $(this).attr('data-content');
         }
     });
-
+  $("#phoneInput").on('click', function() {
+    if ($("#updatePhone").html() == "Save") {
+        $("#updatePhone").html('Edit');
+    } else {
+        $("#updatePhone").html('Save');
+    }
+  });
```

Comment:
This is good but it does not work the way we would want it to in all cases. It performs as we want when we click the box but if we click off the input area it does not change the status of the box. Take a quick look at the course instructor's phone number in SLCs to see an example.

Reply:
base.js already included this functionality in the validatePhoneNumber function, it probably isn't working but I think that would be the place to change it instead. Making a new function probably isn't a great fix as we are duplicating that code.